pkg/console/controllers/route/controller.go

```diff
@@ -110,59 +122,151 @@ func (c *RouteSyncController) sync() error {
 		return nil
 	case operatorsv1.Removed:
 		klog.V(4).Infoln("console is in a removed state: deleting route")
-		return c.removeRoute()
+		if err = c.removeRoute(api.OpenshiftConsoleCustomRouteName); err != nil {
```

Comment:
This is still fine if it 404?  IE, set to `Removed` when the custom route is not in use?

Reply:
If the custom route is not used then the it would just return 404 which `removeRoute()` takes care of. We could add the check for `IsCustomRouteSet()`  in this case